src/deploy.coffee

```diff
@@ -0,0 +1,25 @@
+# Description
+#   runs deployment scripts based on slack messages
+#
+# Commands:
+#   hubot deploy dev - deploys api dev as root
+#
+# Author:
+#   Josh Greenberger
+
+exec = require('child_process').exec
+deployScriptPath = '/home/hubot/DeploymentScripts/hubot'
+authorizedSlackUsers = ['josh.greenberger', 'CircleCI']
+
+module.exports = (robot) ->
+
+  robot.respond /deploy dev ([\w-\.]+)$/i, (msg) ->
+    return unless msg.message.user.name in authorizedSlackUsers
+    deployTag = msg.match[1].toLowerCase()
+    msg.send "Starting triggr_api dev continuous deploy."
+    child = exec "sudo #{deployScriptPath}/dev.deploy_api.sh #{deployTag}", (error, stdout, stderr) ->
+      if stderr
+        robot.logger.error "Error: #{stderr}"
```

Comment:
Does robot.logger go back to slack? Would be kinda nice if it doesn't

Reply:
Nope, it goes to a log file in the hubot directory